### DISCUSSION TITLE
Does anyone know why tests fail now?

### DISCUSSION THREAD

**Discussion by oroulet** [FreeOpcUa/opcua-asyncio#1297] [General]
Looks like flake8 has many comments, but all Mr I merged had tests that passed...
Does anyone knows what is happening here?

**Comment by oroulet** [comment]
running flake8 locally also fails with many warning. I cannot remember we have had that before although all warnings make sense

**Comment by schroeder-** [comment]
It only break on erros E9,F63,F7,F82

**Comment by oroulet** [comment]
anyway I made the switch to ruff and fixed al of them using that ruff --fix stuff. hope it did not break anything
